### PR TITLE
M0078DDG-316 add javadoc references to Appilcation Developper Guide

### PR DESCRIPTION
**Pull request by fpinel-microej**

### CODE REVIEW

**Comment by fpinel-microej** [comment]
Note :  `apply(GraphicsContext g, Size size)` mentioned in ApplicationDeveloperGuide/UI/MWT/how-to-background.rst doesn't seem to exist, there is only `apply(GraphicsContext g, int width, int height)` in `Background` at least https://repository.microej.com/javadoc/microej_5.x/apis/ej/mwt/style/background/Background.html

**Comment by fpinel-microej** [comment]
There doesn't seem to be Javadoc for the Javascript methods, so most of the Javascript doc sections don't have much references for now.

File: ApplicationDeveloperGuide/UI/Advanced/how-to-text-overflow.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ Instrumenting the Widget
 ------------------------
 
 The goal is to check whether the text to be displayed is within the content bounds of the widget. A way to test this is to extend or modify the widget.
-In this article, the widget ``MyLabel`` will extend the type ``Label`` from the Widget library, which displays a text:
+In this article, the widget ``MyLabel`` will extend the type `Label <https://repository.microej.com/javadoc/microej_5.x/apis/ej/widget/basic/Label.html>`_ from the Widget library, which displays a text:
```
Comment (frivieremicroej) [inline ApplicationDeveloperGuide/UI/Advanced/how-to-text-overflow.rst:12]: Please replace links with the shortcut version suggested by @tdelhomenie-microej to improve rst readability

**Review by frivieremicroej — CHANGES_REQUESTED** [review]

**Comment by frivieremicroej** [comment]
@thomasponsmicroej can you comment on https://github.com/MicroEJ/docs/pull/425#issuecomment-1173724017

**Comment by thomasponsmicroej** [comment]
> Note : `apply(GraphicsContext g, Size size)` mentioned in ApplicationDeveloperGuide/UI/MWT/how-to-background.rst doesn't seem to exist, there is only `apply(GraphicsContext g, int width, int height)` in `Background` at least https://repository.microej.com/javadoc/microej_5.x/apis/ej/mwt/style/background/Background.html

I fixed the page (method signature + snippet)

File: ApplicationDeveloperGuide/UI/Advanced/how-to-text-overflow.rst (hunk 1 above)
Comment (fpinel-microej) [reply ApplicationDeveloperGuide/UI/Advanced/how-to-text-overflow.rst]: Done

**Comment by fpinel-microej** [comment]
> > Note : `apply(GraphicsContext g, Size size)` mentioned in ApplicationDeveloperGuide/UI/MWT/how-to-background.rst doesn't seem to exist, there is only `apply(GraphicsContext g, int width, int height)` in `Background` at least https://repository.microej.com/javadoc/microej_5.x/apis/ej/mwt/style/background/Background.html
> 
> I fixed the page (method signature + snippet)

Thank you, I did assume they were the same but wasn't 100% sure so I wanted a second opinion

File: ApplicationDeveloperGuide/classpath.rst [hunk 2]
```diff
@@ -176,13 +183,16 @@ resource. Example:
    # The following resource is embedded as a raw resource
    com/mycompany/MyResource.txt
 
+
+.. _Class.getResourceAsStream(java.io.InputStream): https://repository.microej.com/javadoc/microej_5.x/apis/java/lang/Class.html#getResourceAsStream-java.lang.String-
```
[the remark this answers is not in the document]
Comment (fpinel-microej) [reply ApplicationDeveloperGuide/classpath.rst]: Yes, didn't see this change in the merge conflict resolve, fixed

**Review by frivieremicroej — APPROVED** [review]
Thanks @fpinel-microej !

**Pull request merged**